### PR TITLE
Remove stress tests

### PR DESCRIPTION
**Pull request by rjernst**
These are not actually tests, but command line applications that must be
run manually. This change removes the entire stresstest package. We can
add back individual tests that we find necessary, and make them real
tests (whether integ or not).

### CODE REVIEW

File: core/src/test/java/org/elasticsearch/benchmark/stress/NodesStressBenchmark.java [hunk 1]
```diff
@@ -46,7 +46,7 @@
 /**
  *
  */
-public class NodesStressTest {
```
Comment (nik9000) [inline core/src/test/java/org/elasticsearch/benchmark/stress/NodesStressBenchmark.java]: Should this file be removed too?

File: core/src/test/java/org/elasticsearch/benchmark/stress/NodesStressBenchmark.java [hunk 2]
```diff
@@ -46,7 +46,7 @@
...
+public class NodesStressBenchmark {
```
Comment (dakrone) [inline core/src/test/java/org/elasticsearch/benchmark/stress/NodesStressBenchmark.java]: How come this one passes the gauntlet? :)

**Comment by nik9000** [comment]
Makes sense to me though I'd love to hear from someone else if they think these are valuable or they are using them.

File: core/src/test/java/org/elasticsearch/benchmark/stress/NodesStressBenchmark.java (hunk 2 above)
Comment (rjernst) [inline core/src/test/java/org/elasticsearch/benchmark/stress/NodesStressBenchmark.java]: I had removed just the stresstest package. I will remove benchmark/stress too.

**Comment by dakrone** [comment]
+1 to this, it's frustrating having to continually sync runnable classes that aren't actually ever run, like Nik, I left a comment about the one that you decided not to remove

**Comment by s1monw** [comment]
lets get rid of it

**Pull request merged**